### PR TITLE
Dfe 552 methodology homepage

### PR DESCRIPTION
**Pull request by shakes90**

### CODE REVIEW

File: src/GovUk.Education.ExploreEducationStatistics.Content.Api/appsettings.Development.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "ConnectionStrings": {
-    "AzureStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://data-storage:10000/devstoreaccount1;",
```
Comment (jamescutts) [inline src/GovUk.Education.ExploreEducationStatistics.Content.Api/appsettings.Development.json]: might want to revert "AzureStorage"

File: src/GovUk.Education.ExploreEducationStatistics.Data.Api/appsettings.Development.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "ConnectionStrings": {
-    "StatisticsDb": "Server=db;Database=master;User=SA;Password=Your_Password123;",
-    "AzureStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;"
```
Comment (jamescutts) [inline src/GovUk.Education.ExploreEducationStatistics.Data.Api/appsettings.Development.json]: might want to revert "AzureStorage"

File: src/explore-education-statistics-frontend/src/modules/methodologies/MethodologyPage.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+          <div className="govuk-grid-column-one-third">
+            <PageSearchForm />
+          </div>
+        </div>
+
+        <hr />
+        <div className="govuk-grid-row">
+          <div className="govuk-grid-column-two-thirds">
+            <p className="govuk-body-l">
+              Find out about the methodology behind pupil absence statistics and
```
Comment (jamescutts) [inline src/explore-education-statistics-frontend/src/modules/methodologies/MethodologyPage.tsx]: Should this be data driven?

Comment (shakes90) [reply src/explore-education-statistics-frontend/src/modules/methodologies/MethodologyPage.tsx]: Yep - done cheers

**Pull request merged**